src/overlays/actors/ovl_En_Box/z_en_box.h

```diff
@@ -4,11 +4,84 @@
 #include <ultra64.h>
 #include <global.h>
 
+#define ENBOX_TREASURE_FLAG_UNK_MIN 20
+#define ENBOX_TREASURE_FLAG_UNK_MAX 32
+
 struct EnBox;
 
+typedef void (*EnBoxActionFunc)(struct EnBox*, GlobalContext*);
+
+typedef enum {
+    /*
```

Comment:
is this and the 1FB enum used outside the actor? We only want to keep public things that we are exposing to other files in the header. So if not, they can go in the C file just under the THIS macro definition.
Im thinking type is the only enum that will end up being used externally for params.